src/theme/SearchBar/index.tsx

```diff
@@ -51,9 +55,33 @@ function SearchBar(): JSX.Element | null {
         return () => document.removeEventListener("mousedown", handler);
     }, [dialogRef]);
 
+    useEffect(() => {
+        const isCtrlK = (event: KeyboardEvent) => event.ctrlKey && event.key === 'k';
```

Comment:
This does not support macOS. This may be resolved by changing it to:
`const isCtrlK = (event: KeyboardEvent) => (event.ctrlKey || event.metaKey) && event.key === 'k';`